packer/payload/prepare-ami.sh

```diff
@@ -37,24 +44,25 @@ apt-get install -qy \
 
 apt-mark hold docker-engine
 
+
 ## Install official Kubernetes binaries
 mkdir -p /tmp/kubebin
 (
   cd /tmp/kubebin
   curl -sf -O "https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_RELEASE}/bin/linux/amd64/kubelet"
   curl -sf -O "https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_RELEASE}/bin/linux/amd64/kubectl"
   curl -sf -O "https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_RELEASE}/bin/linux/amd64/kubeadm"
-  curl -sf -O "https://storage.googleapis.com/kubernetes-release/network-plugins/cni-amd64-${CNI_RELEASE}.tar.gz"
+  curl -sf -O "https://storage.googleapis.com/kubernetes-release/network-plugins/${CNI}"
 
   install -o root -g root -m 0755 ./kubeadm /usr/bin/kubeadm
   install -o root -g root -m 0755 ./kubectl /usr/bin/kubectl
   install -o root -g root -m 0755 ./kubelet /usr/bin/kubelet
 
   # Also install CNI
-  mkdir -p /opt/cni
+  mkdir -p /opt/cni/bin
```

Comment:
The new cni tarball does not pack things into `bin/` like the last one used to.

Reply:
If we're going to pay-down debt, I think we should use apt-get/yum/or @craigtracey work.